pyat/at/load/madx.py

```diff
@@ -713,7 +713,7 @@ def beta() -> float:
             if cavities:
                 cavities.sort(key=lambda el: el.Frequency)
                 c0 = cavities[0]
-                params["_harmnumber"] = getattr(c0, "HarmNumber", np.nan)
+                params["_cell_harmnumber"] = getattr(c0, "HarmNumber", np.nan)
```

Comment:
this modification does not seem to belong to the same pull request. is it needed? what is _cell_harmnumber?